src/lib.rs

```diff
@@ -72,6 +76,39 @@ pub enum Class {
     RadioMandatoryZone,
     /// Transponder mandatory zone
     TransponderMandatoryZone,
+    /// OpenAir extension records for French airspace classes.
+    /// These extensions are documented in the following repository:
+    /// https://github.com/BPascal-91/eAirspacesFormats/tree/master/openair/#openair-extended---version-actuelle-%C3%A9tandue-avec-historique-des-%C3%A9volutions-
+    /// NOTAM
+    Notam,
+    /// NOTAM reference
+    NotamRef,
+    /// Zone Sensibilité Majeur
+    Zsm,
+    /// FFVL Protocole for PARAGLIDER
+    Ffvl,
+    /// FFVP Protocole for GLIDER
+    Ffvp,
+    /// Service d'Information en Vol
+    Siv,
+    /// Regulated Air Space
+    Ras,
+    /// Air Defense Identification Zone
+    Adiz,
+    /// Minimum Altitude Area
+    Ama,
+    /// PART of airspace
+    Part,
+    /// Flight Information Region
+    Fir,
+    /// Upper Flight Information Region
+    Uir,
+    /// Oceanic Control Area
+    Oca,
+    /// Political-administrative area
+    Political,
+    /// Airspace for which not even an FIR is defined
+    NoFir,
```

Comment:
wouldn't it be easier to use something like `Other(String)` to cover all of these non-standard values?

side note: the Naviter folks are supporting these kinds of airspace types via `AY` instead (see https://github.com/naviter/seeyou_file_formats/blob/main/OpenAir_File_Format_Support.md#ay-airspace-type)